doc/nrf/releases_and_maturity/release_notes.rst

```diff
@@ -20,7 +20,7 @@ This page is included only in the latest documentation, because it might contain
    :maxdepth: 1
    :caption: Subpages:
 
-   releases/release-notes-changelog
+   releases/release-notes-3.2.0
```

Comment:
The bullet on line no: 15 should be removed. and aspects ->aspect on line no:13.